.github/workflows/release.yaml

```diff
@@ -19,9 +13,26 @@ jobs:
 
   release:
     needs: check
-    uses: canonical/bootstack-actions/.github/workflows/charm-release.yaml@v3
-    secrets: inherit
-    with:
-      channel: "latest/edge"
-      upload-image: false
-      runs-on: "['ubuntu-20.04']"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Initialize lxd  # This should dropped once it's implemented on charming-actions itself. https://github.com/canonical/charming-actions/issues/140
+        if: ${{ inputs.destructive-mode == false }}
+        run: |
```

Comment:
needs to be updated to use the `setup-lxd` action

Reply:
I pushed my changes to my fork instead of directly to this repo. Fixed.